styles/blocks/columns-1.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I believe once we apply the styles in the columns directly we shouldn't need this one.

Reply:
Removed in https://github.com/WordPress/twentytwentyfive/pull/118/commits/a580f0c6ccd69d21d15519c2a191a1a2db413530